api/src/main/java/org/openmrs/OrderGroup.java

```diff
@@ -208,4 +210,25 @@ public void setId(Integer id) {
 		setOrderGroupId(id);
 	}
 	
+	/**
+	 * Gets the parent order group to maintain linkages between groups and support group nesting
+	 * 
+	 * @param returns the parent order group
+	 * @since 2.3.0..
+	 */
+	public OrderGroup getParentOrderGroup() {
+		return parentOrderGroup;
+	}
+	
+	/**
+	 * Sets the ParentOrderGroup to other OrderGroups, to maintain linkages between groups and
```

Comment:
Edit this one like like you have just done for the setter.

Reply:
@dkayiwa i have done the editing of the setter,,requesting for review